src/AAT.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @description PocketAAT entry point.
+ */
+
+import { sha3_256 } from 'js-sha3';
+import ed25519 = require('ed25519');
```

Comment:
Why 2 different import styles?

Reply:
If a library provides a declaration.ts file we can use the import * from style. But if the library doesn't provide the declaration.ts file, we need to use the import variable = require style

Reply:
I'm just concerned about the readability of the file for having 2 different import formats.

Reply:
Ok, I understand, what we can do is, create the declaration file for the ED25519 library and with that file we can use the same import style. Let me know what do you think